cookbooks/fb_profile/files/debian.bashrc

```diff
@@ -0,0 +1,107 @@
+# controlled by Chef, do not edit
```

Comment:
I think this got munged, you should add the standard blurb like we have in the template